lexicons/com/atproto/sync/getRoot.json

```diff
@@ -0,0 +1,34 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.sync.getRoot",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Gets the current root CID of a repo.",
+      "parameters": {
+        "type": "params",
+        "required": ["did"],
+        "properties": {
+          "did": {
+            "type": "string",
+            "format": "did",
+            "description": "The DID of the repo."
+          }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["root"],
+          "properties": {
+            "root": {"type": "string", "format": "cid"}
```

Comment:
would also change this from `root` to `commit`, as this is the CID of the signed commit, not the top node in the MST

Reply:
I decided to change it to `cid` 👍